### PR TITLE
Gungle chickens

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Untamed chickens will now spawn on jungle worlds, they can be fed like a normal chicken, and can be used to raise your own flock

## Why It's Good For The Game
Chickens are cool + it adds to our abilities to forage things

## Changelog

:cl:
add: Chickens will now infrequently spawn on jungle planets
/:cl:

### CODE REVIEW

**Comment by Latentish** [comment]
TMING IMMEDIATELY

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -348,6 +348,94 @@
...
+	var/list/validColors = list("brown","black","white")
+	gold_core_spawnable = FRIENDLY_SPAWN
+	var/static/chicken_count = 0
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	melee_damage_lower = 3
+	melee_damage_upper = 10
+
+	footstep_type = FOOTSTEP_MOB_CLAW
+
+/mob/living/simple_animal/chicken/hostile/retaliate/Initialize()
```
Comment (tmtmtl30) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: the order here is wrong; presumably this has been copypasted from the /mob/living/simple_animal/chicken?

**Review by tmtmtl30 — DISMISSED** [review]

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm (hunk 1 above)
Comment (Erikafox) [reply code/modules/mob/living/simple_animal/friendly/farm_animals.dm]: correct, let me fix that

**Comment by AverageUser67** [comment]
merge it
if spiders and carp attack chickens they should butcher them and eat the meats too
actual ecosystem ftw (i miss basicmob behaviors)

File: code/game/objects/effects/spawners/mobspawner.dm [hunk 2]
```diff
@@ -58,3 +58,22 @@
...
+	loot = list(
+		/mob/living/simple_animal/hostile/retaliate/chicken
+	)
+
+/obj/effect/spawner/lootdrop/chicken/jungle/flock
+	loot = list(
+		/obj/effect/spawner/lootdrop/chicken/jungle
+	)
+	lootcount = 7 //I was going to make this random but it wasn't cooperating
+	lootdoubles = TRUE
```
Comment (MarkSuckerberg) [inline code/game/objects/effects/spawners/mobspawner.dm]:
```suggestion
/obj/effect/spawner/lootdrop/chicken/jungle/flock
	loot = list(
		/obj/effect/spawner/lootdrop/chicken/jungle
	)
	lootcount = 7
	lootdoubles = TRUE

/obj/effect/spawner/lootdrop/chicken/jungle/flock/New()
	lootcount = rand(5, 10)
```

Comment (Erikafox) [reply code/game/objects/effects/spawners/mobspawner.dm]: Ough

**Review by MarkSuckerberg — APPROVED** [review]
birgrt :+1:

**Pull request merged**